### PR TITLE
Fixes #11: provide an example of a demo deployment of secrets-manager

### LINKED ISSUE #11: Provide Deployment Examples
In order to make easier to understand how _secrets-manager_ works and, it would be nice having an example inside an `examples/` folder showing how to configure and deploy it.

### PR DESCRIPTION
**Pull request by fllaca**
Added a simple deployment example using K8s manifests. It also includes a dummy deployment of Vault.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+kubectl apply -f vault.yaml
+```
+
+## Generate a Vault token 
+... and store it in a K8s secret. If you are going to use your current Vault instance you don't need the first line, just generate a Vault token as usual and go straight to second line to create it.
+
+```bash
+VAULT_TOKEN=$(kubectl exec $(kubectl get pods -l app=vault -o custom-columns=:metadata.name) -c vault -- vault token create --field=token)
+
+kubectl	create secret generic --from-literal=token=$VAULT_TOKEN vault-token-secret
```
Comment (fcgravalos) [inline examples/README.md]: remove extra tab

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+```
+
+```
+➜  kubectl get secrets
+NAME                  TYPE                                  DATA      AGE
+default-token-9k5dv   kubernetes.io/service-account-token   3         34d
+supersecret1          kubernetes.io/tls                     2         22h
+supersecret2          Opaque                                2         22h
+vault-token-secret    Opaque                                1         22h
+
```
Comment (fcgravalos) [inline examples/README.md]: remove extra line

File: examples/README.md (hunk 1 above)
Comment (fllaca) [reply examples/README.md]: done, thx!

File: examples/README.md (hunk 2 above)
Comment (fllaca) [reply examples/README.md]: done!

**Pull request merged**